app.rb

```diff
@@ -1,20 +1,129 @@
 require_relative "data"
+require_relative "models/apartment"
+require_relative "models/tenant"
 apartments = data[:apartments]
 tenants = data[:tenants]
 
+ruby_apartments = []
+ruby_tenants = []
+#id, address, monthly_rent, square_feet
+ruby_apartments.each do |unit|
+  unit = Apartment.new(unit[:id], unit[:address], unit[:monthly_rent], unit[:square_feet])
+  apartments << ruby_apartments
```

Comment:
This line should be:

```ruby
ruby_apartments << unit
```

because you want to *add* the unit you just created *to* the empty array called `ruby_apartments`